Makefile

```diff
@@ -50,7 +50,7 @@ lint :
 
 .PHONY : format
 format :
-	black --check .
+	black --exclude training_config_test.py --check .
```

Comment:
We could use

```python

# fmt: off
...
# fmt: on
```

comments within the code instead. I think that would be a better solution since many people might have black run automatically within their code editor (like me). Otherwise I would have to remember to turn it off everytime I edit this file.

Reply:
I'll make the change.